examples/with-next-js/pages/partytown/index.tsx

```diff
@@ -44,7 +44,7 @@ const WebWorker: React.FC = () => {
           }}
           value="Track!"
```

Comment:
It's actually fine to do:
```
declare global {
  interface Window {
    analytics: AnalyticsSnippet
  }
}
```
... since this is not a published repo

Reply:
I'll leave it as it is for now. But yeah, doesn't matter.